### PR TITLE
[Ultica] Sprites for boxes

### PR DESCRIPTION
**Pull request by GuakeTheAcinid**
#### Summary

Added sprites for different types of boxes

#### Content of the change
item sprites:
- tiny wooden box
- small wooden box
- small plastic box
- compact wooden box
- medium wooden box
- large woodern box
- oversize wooden box
- military black box

human_body wielded overlay sprites:
- medium wooden box
- medium metal box
- small wooden box
- small metal box
- small plastic box
- military black box

#### Testing
testToShowEverythingIsAlright

### CODE REVIEW

File: gfx/UltimateCataclysm/pngs_human_body_32x36/overlay/wielded/by_Guake/overlay_wielded_black_box.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "id": [ "overlay_wielded_black_box" ],
```
Comment (Fris0uman) [inline gfx/UltimateCataclysm/pngs_human_body_32x36/overlay/wielded/by_Guake/overlay_wielded_black_box.json:2]: unless `overlay_wielded_black_box` is also used in json somewhere else this is unecessary. Naming the file with the correct id is enough if you want that sprite to be used just for this id

File: gfx/UltimateCataclysm/pngs_human_body_32x36/overlay/wielded/containers/overlay_wielded_box.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      "fg": [ "overlay_wielded_box_medium_metal" ]
+    },
+    {
+      "id": [ "overlay_wielded_box_large_wood", "overlay_wielded_box_medium_wood", "overlay_wielded_box_oversize_wood" ],
+      "fg": [ "overlay_wielded_box_medium_wood" ]
+    },
+    {
+      "id": [ "overlay_wielded_box_small_metal" ],
+      "fg": [ "overlay_wielded_box_small_metal" ]
+    },
```
Comment (Fris0uman) [inline gfx/UltimateCataclysm/pngs_human_body_32x36/overlay/wielded/containers/overlay_wielded_box.json:13]:
```suggestion
```
You don't this

File: gfx/UltimateCataclysm/pngs_human_body_32x36/overlay/wielded/containers/overlay_wielded_box.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    {
+      "id": [ "overlay_wielded_box_small_wood", "overlay_wielded_box_compact_wood" ],
+      "fg": [ "overlay_wielded_box_small_wood" ]
+    },
+    {
+      "id": [ "overlay_wielded_box_small_plastic" ],
+      "fg": [ "overlay_wielded_box_small_plastic" ]
+    }
```
Comment (Fris0uman) [inline gfx/UltimateCataclysm/pngs_human_body_32x36/overlay/wielded/containers/overlay_wielded_box.json:21]:
```suggestion
```
same

File: gfx/UltimateCataclysm/pngs_normal_32x32/items/containers/box_wood.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+[
+    {
```
Comment (Fris0uman) [inline gfx/UltimateCataclysm/pngs_normal_32x32/items/containers/box_wood.json:2]: unnecessary

File: gfx/UltimateCataclysm/pngs_small_20x20/items/containers/box_small_plastic.json [hunk 5]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (Fris0uman) [inline gfx/UltimateCataclysm/pngs_small_20x20/items/containers/box_small_plastic.json:1]: don't need it

File: gfx/UltimateCataclysm/pngs_small_20x20/items/containers/box_wood.json [hunk 6]
```diff
@@ -0,0 +1,10 @@
+[
+    {
+      "id": [ "box_small_wood" ],
```
Comment (Fris0uman) [inline gfx/UltimateCataclysm/pngs_small_20x20/items/containers/box_wood.json:3]: don't need it

**Review by Fris0uman — CHANGES_REQUESTED** [review]
A few fix needed:
- Remove the unecessary json
- don't put your name on folders, name them something sensible like here `overlay_wielded_balc_box`  and `black_box` for exemple

**Comment by GuakeTheAcinid** [comment]
@Fris0uman, fixes are made.

**Review by Fris0uman — APPROVED** [review]

**Pull request merged**